src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostClaimByIdControllerCFTTest.java

```diff
@@ -115,15 +118,28 @@ public void should_return_a_204_when_claiming_a_task_by_id() {
             .statusCode(HttpStatus.NO_CONTENT.value());
 
         assertions.taskVariableWasUpdated(taskVariables.getProcessInstanceId(), "taskState", "assigned");
-        assertions.taskStateWasUpdatedInDatabase(taskId, "assigned", caseworkerCredentials.getHeaders());
-        String serviceToken = caseworkerCredentials.getHeaders().getValue(AUTHORIZATION);
-        UserInfo userInfo = authorizationProvider.getUserInfo(serviceToken);
-        assertions.taskFieldWasUpdatedInDatabase(
+
+        result = restApiActions.get(
+            "/task/{task-id}",
             taskId,
-            "assignee",
-            userInfo.getUid(),
             caseworkerCredentials.getHeaders()
         );
+
```

Comment:
Please add this test in watasks claim function test as we are going to get rid of IAC tests